README.md

```diff
@@ -153,16 +172,6 @@ target 'MyAppTests' do
 end
 ```
 
-### Swift Package Manager
```

Comment:
Also moved Carthage and CocoaPods below the fold to start with first-party tooling.